crates/telio-firewall/src/firewall.rs

```diff
@@ -310,6 +319,12 @@ struct TcpConn {
     pubkey: PublicKey,
 }
 
+#[derive(Debug, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
+struct UdpConn {
+    link: IpConnWithPort,
+    pubkey: PublicKey,
```

Comment:
Interestingly enough, I think adding key here covers LLT-4502 for UDP, and TCP is already covered by previous PR. Thus in order to finish LLT-4502, only ICMP seems to be left :).

Reply:
Yep, correct. That's why I marked LLT-4502 as blocked by this feature